presto-parser/src/test/java/io/prestosql/sql/parser/TestSqlParserErrorHandling.java

```diff
@@ -39,85 +39,85 @@
     {
         return new Object[][] {
                 {"",
-                        "line 1:1: mismatched input '<EOF>'. Expecting: 'ALTER', 'ANALYZE', 'CALL', 'COMMIT', 'CREATE', 'DEALLOCATE', 'DELETE', 'DESC', 'DESCRIBE', 'DROP', 'EXECUTE', 'EXPLAIN', 'GRANT', " +
```

Comment:
While I agree that this looks nicer in this circumstance, it does not match the code formatter. To double check, I verified against the formatter settings in the codestyle repo, and the results were the same as the current committed version.

This result makes sense, as the formatter is treating the second item as a continuation indent. Putting the first item on the same line as the brace is a special case. Instead, consider the more general case:
```java
return new Object[][] {
        {
                "abc",
                "xyz",
        },
        {
                "foo",
                "bar",
        },
};
```

I also couldn't find any formatter setting that matches your proposed change.

Reply:
I just reformatted the code with IntelliJ and that’s what I got. That’s odd.

Reply:
I found the setting that causes that. It's "align when multiline" under "array initializer" formatting.

Reply:
That settings seems inconsistent with everything else, as we don't align method parameters, etc. For example, it results in this:
```java
int[] abc = new int[] {1, 2,
                       3, 4};
```
If we're going to start using that style, we need to update the codestyle and reformat the code base to match.

Reply:
If that was set in the codestyle, it would look like this:
```xml
<codeStyleSettings language="JAVA">
  <option name="ALIGN_MULTILINE_ARRAY_INITIALIZER_EXPRESSION" value="true" />
</codeStyleSettings>

Reply:
Yeah, I’ll remove that commit. We should consider adopting that style for arrays, though. It’s a little different from method parameters, as arrays contain homogeneous elements and the visual structure can help when there are lots of elements. Clearly we don’t want to force one element per line like we do for method args.